gammapy/data/event_list.py

```diff
@@ -132,7 +132,14 @@ def altaz(self):
 
         lon, lat = self['AZ'], self['ALT']
         return SkyCoord(lon, lat, unit='deg', frame=altaz_frame)
-
+    
+    @property
+    def offset(self):
```

Comment:
It's better if you replace the implementation of the `offset` computation with this:
https://github.com/gammapy/gammapy/blob/master/gammapy/data/event_list.py#L265

I.e. use the RADEC pointing and event position and angular separation on the sky.
I prefer this for two reasons:
- The DETX, DETY field of view coordinates aren't well-defined yet, see [here](http://gamma-astro-data-formats.readthedocs.org/en/latest/info/coordinates.html#field-of-view)
- The FOV coordinates are optional, whereas RA / DEC are required and thus always present (see [here](http://gamma-astro-data-formats.readthedocs.org/en/latest/events/index.html#required-column-names)).

Angular separation on the sky will be slower, but I don't think performance is an issue here.
OK?